pyproject.toml

```
@@ -0,0 +1,34 @@
+[build-system]
+requires = ["setuptools", "setuptools-scm"]
+
+
+[tool.setuptools]
+packages = ["usd_qtpy"]
+
+
+[project]
+name = "usd-qtpy"
+authors = [
+    {name = "BigRoy"},
+    {name = "Hannes"},
+]
+description = "Python Qt components for building custom USD tools."
+readme = "README.md"
+requires-python = ">=3.4"
+keywords = ["usd", "qt"]
+license = { file = "LICENSE" }
+classifiers = [
+    "Programming Language :: Python :: 3.7",
+]
+dependencies = [
+    'importlib-metadata; python_version<"3.7"', "qtpy"
+]
+version = "0.0.1"
+
+[project.optional-dependencies]
+usd = ["usd-core"]
+usdview = ["PyOpenGL"]
+
+[project.urls]
+Homepage = "https://github.com/BigRoy/usd-qtpy"
+Documentation = "https://github.com/BigRoy/usd-qtpy"
```

Should we include issues in the list? I noticed that being used in the example [here](https://packaging.python.org/en/latest/tutorials/packaging-projects/#configuring-metadata).
```suggestion
Documentation = "https://github.com/BigRoy/usd-qtpy"
Issues = "https://github.com/BigRoy/usd-qtpy/issues"
```